### PR TITLE
Add dark_grey alias

### PR DESCRIPTION
**Pull request by Andre601**
Adds the `dark_grey` alias for `dark_gray` (Completely forgot to mention that this one also exists... oops)

### CODE REVIEW

File: src/main/java/net/kyori/adventure/text/minimessage/transformation/inbuild/ColorTransformation.java [hunk 1]
```diff
@@ -51,6 +51,7 @@
   private static Map<String, String> colorAliases = new HashMap<>();
 
   static {
+    colorAloases.put("dark_grey", "dark_gray");
```
Comment (kezz) [inline src/main/java/net/kyori/adventure/text/minimessage/transformation/inbuild/ColorTransformation.java:54]:
```suggestion
    colorAliases.put("dark_grey", "dark_gray");
```

**Comment by MiniDigger** [comment]
gotta love web editor PRs :D
thanks anyways

**Comment by Andre601** [comment]
> gotta love web editor PRs :D
> thanks anyways

Why that?

**Comment by MiniDigger** [comment]
just generally

**Pull request merged**